chsdi/locale/en/LC_MESSAGES/chsdi.po

```diff
@@ -1086,7 +1086,7 @@ msgid "ch.bafu.bundesinventare-amphibien_anhang4"
 msgstr "Amphibians appendix 4"
 
 msgid "ch.bafu.bundesinventare-amphibien_anhoerung"
-msgstr "Amphibien - Anhörung 2015"
```

Comment:
yep, it exists and the DB should be OK.

@alcapat could you please translate anew and check?